requirements.txt

```diff
@@ -1,4 +1,4 @@
-bioread
+bioread==1.0.4
```

Comment:
Any reason we're pinning this as *exactly* equal rather than a `>=`?

Reply:
Compatibility, mainly.
As an alternative, we can change it to bioread >= 1.0.4 < 2.0.0

Reply:
Ooh, what about the code in `bioread >= 2.0.0` breaks the current pipeline? I generally don't like having `<` in dependencies, but if it's unavoidable then so be it.

(To answer your question, though: yes, I would definitely prefer `bioread>=1.0.4,<2.0.0` to what we have now!)

Reply:
I don't know, bioread 2.0.0 hasn't been written yet! :smile: 
However if the dev uses semantic versioning, we can expect it to break, no?
Actually, the same goes for numpy  - we should be able to remove pandas dependencies!

Reply:
Right, but you don't want to preempt these things, generally speaking. If any of the dependencies releases a breaking change then tests will begin to fail on the `phys2bids` master branch and you can release a new version of `phys2bids` with updates to address the broken changes.

For now I would say just do `>=` for all these dependencies so we have a good starting point (i.e., we know that, as of right now, people will have the minimum required versions of the packages so that `phys2bids` works). We can address any breaking changes as they arise in the future.

Reply:
I see. Ok, no problem!